content/docs/0.8.x/integrations/migrate_from_070_to_080/index.md

```diff
@@ -0,0 +1,739 @@
+---
+title: Migrate existing Keptn integration
+description: Technical guidance regarding migrating an existing Keptn 0.7.x service to a Keptn 0.8.x service
```

Comment:
Technical guidance **to migrate** an existing Keptn 0.7.x service to a Keptn 0.8.x service